plugins/node/plugin.zsh

```diff
@@ -5,14 +5,18 @@ GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
 GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
 GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
 
-geometry_prompt_node_setup() {}
+geometry_prompt_node_setup() {
+    (( $+commands[node] )) || return 1
+}
+
+geometry_prompt_node_check() {
+    test -f package.json || return 1
+}
 
 geometry_prompt_node_render() {
-    if (( $+commands[node] )); then
-        GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
-        GEOMETRY_NPM_VERSION="$(npm --version 2> /dev/null)" 
-        echo "$GEOMETRY_NODE_NPM_VERSION $GEOMETRY_NODE_VERSION ($GEOMETRY_NPM_VERSION)"
-    fi
+    GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
```

Comment:
Just a note: don't forget to update this with the code recently merged from #108